Src/BootCamp.Chapter1/ArrayOperations.cs

```diff
@@ -75,8 +174,34 @@ public static int[] InsertFirst(int[] array, int number)
         /// <returns>A new array with element added in the end of array. If an array is empty or null, returns new array with number in it.</returns>
         public static int[] InsertLast(int[] array, int number)
```

Comment:
What does this have in common with InsertAt? :)

Reply:
return IfArrayNull(array, number);

return ArrayLengthZero(array, number);